rfcs/supported/algorithms_histogram_cpu_backends/README.md

```diff
@@ -138,38 +138,94 @@ percentage of the output histogram may be occupied, even when considering dividi
 This could be explored if we find temporary storage is too large for some cases and the atomic approach does not
 provide a good fallback.
 
-## Proposal
-After exploring the above implementation for `histogram`, the following proposal better represents the use cases which
-are important, and provides reasonable performance for most cases.
+## Supported Implementation
+After exploring the above implementation for `histogram`, the following implementation better represents the use cases
+which are important, and provides reasonable performance for most cases.
+
+### Version Information
+Histogram APIs are supported for host execution policies as of version 2022.8.0 of oneDPL, specification version 1.4.
```

Comment:
It would be great to found the version information easily at the beginning of the document.
Also I would move up the chapter describing the implementation above the thoughts on alternatives, which are now mostly of historical interest.

Reply:
moved the sections around toward this end.